### PR TITLE
Added unit tests for 1 component (textAnswers) using Vitest and React Testing Library

### PR DESCRIPTION
**Pull request by kobebuckley**
Initial setup for Vitest & React Testing Library

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,11 +11,12 @@
...
     "prepare": "husky install",
     "prettier": "prettier --write .",
-    "prettier:ci": "prettier --check ."
+    "prettier:ci": "prettier --check .",
+    "test": "vitest"
   },
   "dependencies": {
     "@next-auth/prisma-adapter": "^1.0.5",
-    "@prisma/client": "^4.10.0",
+    "@prisma/client": "4.10.0",
```
Comment (avenmia) [inline package.json:19]: The ^ is removed here, can you add it back in?

File: src/components/InfoPopup.tsx [hunk 2]
```diff
@@ -48,7 +48,7 @@ const InfoPopup = ({ title, PopupInfo }: InfoPopupProps) => {
         >
           <button
             onClick={toggleMoreInformation}
-            className="absolute top-4 right-4 text-2xl lg:text-4xl"
+            className="absolute right-4 top-4 text-2xl lg:text-4xl"
```
Comment (avenmia) [inline src/components/InfoPopup.tsx:51]: Can you go through and switch these back to their original style order? If it's a prettier issue we can do another prettier PR. The unit tests should all still pass

**Review by avenmia — CHANGES_REQUESTED** [review]

File: src/components/InfoPopup.tsx (hunk 2 above)
Comment (kobebuckley) [reply src/components/InfoPopup.tsx]: I added  /* prettier-ignore */

above that line so now it won't be affected by prettier.

**Review by avenmia — APPROVED** [review]
lgtm! 👍

**Pull request merged**